blueprints/authenticator/index.js

```diff
@@ -15,25 +15,25 @@ module.exports = {
 
     if (baseClass === 'torii') {
       return {
-        imports: 'import Ember from \'ember\';' + EOL + 'import Torii from \'ember-simple-auth/authenticators/torii\';',
+        imports: 'import Torii from \'ember-simple-auth/authenticators/torii\';',
         baseClass: 'Torii',
         body: EOL + '  torii: Ember.inject.service(\'torii\')'
```

Comment:
I've not actually tested this, but just happened to notice that this authenticator does appear to use the imported Ember module.